draft-pda-protocol.md

```diff
@@ -479,13 +475,6 @@ Client          Leader         Helper
 ~~~~
 {: #pa-upload-flow title="Flow of the upload process"}
 
-[NOTE: @acmiyaguchi pointed out that the use of an anonymizing proxy for
-uploading shares might be easier to implement if the "upload" phase involved a
-single HTTP request. However, OHTTP
-(https//www.ietf.org/archive/id/draft-thomson-http-oblivious-01.html) allows
-clients to make multiple requests through a proxy, so these kinds of use cases
-should work.]
```

Comment:
We have a section in security considerations about OHTTP and anonymizing proxies so I think this is settled.